Checkbox/Checkbox.js

```diff
@@ -30,11 +34,11 @@ import css from './Checkbox.module.less';
  * <Checkbox selected />
  * ```
  *
- * @class Checkbox
+ * @class CheckboxBase
```

Comment:
Since `@class` is changed to `CheckboxBase`, `@lends` comment also need to be updated.
> /** @lends agate/Checkbox.CheckboxBase.prototype */